postcss.config.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   plugins: {
+    'postcss-mixins': {},
+    'postcss-simple-vars': {},
+    'postcss-import': {},
+    'tailwindcss/nesting': {},
     tailwindcss: {},
     autoprefixer: {},
+    cssnano: {},
```

Comment:
We might in the future consider applying this plugin only if ENV is `production` if our CSS becomes too big, due to performance implications.